packit_service/worker/reporting.py

```diff
@@ -87,6 +88,26 @@ def __set_pull_request_status(
                 uid=hashlib.md5(check_name.encode()).hexdigest(),
             )
 
+    def report_status_by_comment(
+        self,
+        state: CommitStatus,
+        url: str,
+        check_names: Union[str, list, None],
+    ):
+        """
+        Reporting build status with MR comment if no permission to the fork project
+        """
+
+        header = ["| Type | Status |", "| ---- | ----- |"]
+
+        if isinstance(check_names, str):
+            check_names = [check_names]
+        comment_msg = [
+            f"| [{check}]({url}) | {state.name.upper()} |" for check in check_names
+        ]
+
+        self.project.pr_comment(self.pr_id, "\n\n".join(header + comment_msg))
```

Comment:
I think we need only one new line here:
```suggestion
        self.project.pr_comment(self.pr_id, "\n".join(header + comment_msg))
```